lib/prototype-rails/selector_assertions.rb

```diff
@@ -126,9 +126,15 @@ def assert_select_rjs(*args, &block)
     if matches
       assert true # to count the assertion
       if block_given? && !([:remove, :show, :hide, :toggle].include? rjs_type)
+        to_select = if ActionPack::VERSION::STRING > "4.2.0"
+          Nokogiri::HTML(matches.map(&:to_s).join)
```

Comment:
Does this gem depends on Nokogiri? I don't think so. If we want to use nokogiri we need to make it a dependency